addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4961,7 +4970,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#12362"
-msgid "Shutdown system if Master Lock retries exceeded"
+msgid "Shutdown system if number of \"Master lock\" retries is exceeded"
```

Comment:
No they are not, they serve a purpose, which is to justify and separate the wrong capitalization of second words.

They stay.